index.js

```diff
@@ -292,6 +302,29 @@ module.exports = {
   },
 
   testemMiddleware: function(app) {
+    // `ember test` can be run in two ways that we cater to.
+    // 1) `ember test` can be run WITHOUT the `--path` flag, in which case ember test first runs the
+    // ember build, and then runs the tests.  In this scenario outputReady would have already been
+    // invoked, calling createPercyBuild.
+    // 2) `ember test` can be supplied a `--path=dist` flag, which means the app has been pre-built
+    // in a separate build step, and is being passed in with the `--path` flag.  In this scenario,
+    // outputReady is not invoked by the `ember test` command, so we need to create the Percy build
+    // here.
+    //
+    // The value passed to --path is available in process.env.EMBER_CLI_TEST_OUTPUT
+    // We can also check we're executing the test command with process.env.EMBER_CLI_TEST_COMMAND
+
+    if (!this.createPercyBuildInvoked) {
+      if (process.env.EMBER_CLI_TEST_COMMAND === 'true' && process.env.EMBER_CLI_TEST_OUTPUT) {
+        this.createPercyBuild(process.env.EMBER_CLI_TEST_OUTPUT);
+      } else {
+        console.warn(
```

Comment:
I can't think of a scenario of how we might get here, but lets warn and disable Percy if we do.